pkg/cephfs/volumeoptions.go

```diff
@@ -268,7 +269,7 @@ func newVolumeOptionsFromVersion1Context(volID string, options, secrets map[stri
 			return nil, nil, err
 		}
 
-		opts.RootPath = getVolumeRootPathCeph(volumeID(volID))
+		opts.RootPath = path.Join("/csi-volumes", string(volumeID(volID)))
```

Comment:
is this path configurable or hardcoded?

Reply:
This is the old volume type, its hardcoded and not configurable. In backward compatibility patch have added getVolumeRootPathCephDepricated